### PR TITLE
Add more regression parallelization

### PR DESCRIPTION
**Pull request by furkansahin**
This PR includes the changes related to the regression parallelization PART 2

### CODE REVIEW

File: src/test/regress/multi_schedule [hunk 1]
```diff
@@ -181,8 +163,9 @@ test: multi_unsupported_worker_operations
 
 # ----------
 # multi_schema_support makes sure we can work with tables in schemas other than public with no problem
+# multi_truncate tests truncate functionality for distributed tables
 # ----------
-test: multi_schema_support
+test: multi_schema_support
```
Comment (metdos) [inline src/test/regress/multi_schedule]: minor. unnecessary empty space

File: src/test/regress/multi_schedule [hunk 2]
```diff
@@ -129,33 +127,17 @@ test: multi_outer_join
...
-test: multi_simple_queries
-test: multi_create_insert_proxy
-test: multi_data_types
-test: multi_utilities
-test: multi_repartition_udt
-test: multi_repartitioned_subquery_udf
-test: multi_modifying_xacts
-test: multi_transaction_recovery
-test: multi_subtransactions
+test: multi_generate_ddl_commands multi_create_shards multi_prune_shard_list multi_repair_shards multi_upsert multi_simple_queries multi_create_insert_proxy multi_data_types
```
Comment (metdos) [inline src/test/regress/multi_schedule]: This one is very long. Is one of them runs very long? Can we split this into two?

File: src/test/regress/multi_schedule [hunk 3]
```diff
@@ -80,9 +80,7 @@ test: multi_join_order_tpch_large
 # new shards before these tests, as they expect specific shard identifiers in
 # the output.
 # ----------
-test: multi_large_table_join_planning
-test: multi_large_table_pruning
-test: multi_large_table_task_assignment
+test: multi_large_table_join_planning multi_large_table_pruning multi_large_table_task_assignment
```
Comment (furkansahin) [inline src/test/regress/multi_schedule]: these tests are read-only, so even if they use the same tables, they do not cause a race-condition.

File: src/test/regress/multi_schedule [hunk 4]
```diff
@@ -129,33 +127,17 @@ test: multi_outer_join
 # is independed from the rest of the group, it is added to increase parallelism.
 # ---
 test: multi_create_fdw
-test: multi_complex_count_distinct multi_select_distinct
+test: multi_complex_count_distinct multi_select_distinct multi_modifications
```
Comment (furkansahin) [inline src/test/regress/multi_schedule]: no common tables

File: src/test/regress/multi_schedule (hunk 2 above)
Comment (metdos) [reply src/test/regress/multi_schedule]: lets do that

File: src/test/regress/multi_schedule [hunk 5]
```diff
@@ -129,33 +127,17 @@ test: multi_outer_join
...
-test: multi_data_types
-test: multi_utilities
-test: multi_repartition_udt
-test: multi_repartitioned_subquery_udf
-test: multi_modifying_xacts
-test: multi_transaction_recovery
-test: multi_subtransactions
+test: multi_generate_ddl_commands multi_create_shards multi_prune_shard_list multi_repair_shards multi_upsert multi_simple_queries multi_create_insert_proxy multi_data_types
+test: multi_utilities multi_modifying_xacts
+test: multi_repartition_udt multi_repartitioned_subquery_udf multi_transaction_recovery multi_subtransactions
```
Comment (furkansahin) [inline src/test/regress/multi_schedule]: multi_transaction_recovery tests rely on pg_dist_transaction table but this did not create any problem

**Comment by metdos** [comment]
looks good 🚢

**Pull request merged**